src/features/devices/light/insolation_sensor.py

```diff
@@ -0,0 +1,58 @@
+from features.devices.device import Device, DeviceConfig, Topic
+from features.logger.logger import Logger
+from features.mqtt.mqtt import BaseMqttClient
+from utime import ticks_ms, ticks_diff, time
+from ujson import dumps, loads
+from features.analog_accessor.analog_accessor import AnalogAccessor
+import machine
+from drivers.insolation import InsolationDriver
+import uasyncio
+
+
+class Insolation(Topic):
+    pass
+
+
+class InsolationSensor(Device):
```

Comment:
Tutaj można by dorzucić jeszcze zliczanie jak długo nasloneczniona roślinka była przez ostatnie 24h- ale to tez raczej na nastepny featurek